frontend/src/metabase/public/components/EmbedFrame/EmbedFrame.tsx

```diff
@@ -127,7 +127,7 @@ export const EmbedFrame = ({
     .filter(Boolean)
     .join(",");
 
-  const showFooter = hasEmbedBranding || (downloadsEnabled && actionButtons);
+  const showFooter = hasEmbedBranding || downloadsEnabled || actionButtons;
```

Comment:
@deniskaber this will break public/static questions with #downloads=false
<img width="1125" alt="image" src="https://github.com/user-attachments/assets/a9ec65bf-3020-4877-8c75-d75702fbc9a1">